src/Simplex/Chat.hs

```diff
@@ -837,21 +838,22 @@ processChatCommand = \case
       withStore' $ \db -> deletePendingContactConnection db userId chatId
       pure $ CRContactConnectionDeleted user conn
     CTGroup -> do
-      Group gInfo@GroupInfo {membership} members <- withStore $ \db -> getGroup db user chatId
+      Group gInfo@GroupInfo {membership} members <-
+        withStoreCtx (Just "APIDeleteChat CTGroup, getGroup") $ \db -> getGroup db user chatId
       let isOwner = memberRole (membership :: GroupMember) == GROwner
           canDelete = isOwner || not (memberCurrent membership)
       unless canDelete $ throwChatError $ CEGroupUserRole gInfo GROwner
-      filesInfo <- withStore' $ \db -> getGroupFileInfo db user gInfo
+      filesInfo <- withStoreCtx' (Just "APIDeleteChat CTGroup, getGroupFileInfo") $ \db -> getGroupFileInfo db user gInfo
```

Comment:
why do we need to use withStoreCtx everywhere?

Reply:
it's not necessary